internal/crypto/crypto.go

```diff
@@ -62,6 +64,19 @@ func GenerateRandomBytes(length int) ([]byte, error) {
 	return nonce, nil
 }
 
+// GenerateEmergencySSHCAKey creates a CA that is used to sign keys for emergency ssh access.
+func GenerateEmergencySSHCAKey(key []byte) (ssh.Signer, error) {
+	_, priv, err := ed25519.GenerateKey(bytes.NewReader(key))
```

Comment:
```suggestion
func GenerateEmergencySSHCAKey(seed []byte) (ssh.Signer, error) {
	_, priv, err := ed25519.GenerateKey(bytes.NewReader(seed))
```
nit: just for clarity. The `ed25519` also refers to this input as seed